Publish @golevelup/nestjs-rabbitmq

Any chance for a rabbitmq release soon so that I can start playing around with message batching 🙂 

@WonderPanda @underfisk

I'm trying to publish but running into some issues with the new PNPM setup.

```
pnpm lerna publish -m 'chore: publish'
lerna notice cli v8.1.8
lerna info versioning independent
lerna info Looking for changed packages since @golevelup/nestjs-rabbitmq@5.5.0
lerna info getChangelogConfig Successfully resolved preset "conventional-changelog-angular"

Changes:
 - rabbitmq-integration: 3.4.0 => 3.5.0 (private)
 - @golevelup/nestjs-common: 2.0.0 => 2.0.1
 - @golevelup/nestjs-discovery: 4.0.1 => 4.0.2
 - @golevelup/nestjs-graphql-request: 0.2.2 => 0.2.3
 - @golevelup/nestjs-hasura: 3.0.7 => 3.0.8
 - @golevelup/nestjs-rabbitmq: 5.5.0 => 5.6.0
 - @golevelup/nestjs-stripe: 0.8.2 => 0.9.0
 - @golevelup/ts-jest: 0.5.6 => 0.6.0
 - @golevelup/ts-sinon: 0.1.0 => 0.1.1
 - @golevelup/ts-vitest: 0.5.0 => 0.5.1
 - @golevelup/nestjs-webhooks: 0.2.18 => 0.2.19

? Are you sure you want to publish these packages? Yes
lerna info execute Skipping releases
lerna ERR! Error: Command failed with exit code 1: pnpm install --lockfile-only --ignore-scripts
lerna ERR! Scope: all 13 workspace projects
lerna ERR!  WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
lerna ERR! Progress: resolved 1, reused 0, downloaded 0, added 0
lerna ERR! /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request:
lerna ERR!  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @golevelup/nestjs-discovery@^4.0.2
lerna ERR! 
lerna ERR! This error happened while installing a direct dependency of /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request
lerna ERR! 
lerna ERR! The latest release of @golevelup/nestjs-discovery is "4.0.1".
lerna ERR! 
lerna ERR! If you need the full list of all 12 published versions run "$ pnpm view @golevelup/nestjs-discovery versions".
lerna ERR!     at makeError (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/lib/error.js:59:11)
lerna ERR!     at handlePromise (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/index.js:114:26)
lerna ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
lerna ERR!     at async VersionCommand.updatePackageVersions (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/lerna@8.1.8_encoding@0.1.13/node_modules/lerna/dist/index.js:9857:11)
lerna ERR!     at async Promise.all (index 0)
lerna ERR! Error: Command failed with exit code 1: pnpm install --lockfile-only --ignore-scripts
lerna ERR! Scope: all 13 workspace projects
lerna ERR!  WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
lerna ERR! Progress: resolved 1, reused 0, downloaded 0, added 0
lerna ERR! /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request:
lerna ERR!  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @golevelup/nestjs-discovery@^4.0.2
lerna ERR! 
lerna ERR! This error happened while installing a direct dependency of /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request
lerna ERR! 
lerna ERR! The latest release of @golevelup/nestjs-discovery is "4.0.1".
lerna ERR! 
lerna ERR! If you need the full list of all 12 published versions run "$ pnpm view @golevelup/nestjs-discovery versions".
lerna ERR!     at makeError (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/lib/error.js:59:11)
lerna ERR!     at handlePromise (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/index.js:114:26)
lerna ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
lerna ERR!     at async VersionCommand.updatePackageVersions (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/lerna@8.1.8_encoding@0.1.13/node_modules/lerna/dist/index.js:9857:11)
lerna ERR!     at async Promise.all (index 0)
lerna ERR! lerna Command failed with exit code 1: pnpm install --lockfile-only --ignore-scripts
lerna ERR! lerna Scope: all 13 workspace projects
lerna ERR! lerna  WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
lerna ERR! lerna Progress: resolved 1, reused 0, downloaded 0, added 0
lerna ERR! lerna /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request:
lerna ERR! lerna  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @golevelup/nestjs-discovery@^4.0.2
lerna ERR! lerna 
lerna ERR! lerna This error happened while installing a direct dependency of /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request
lerna ERR! lerna 
lerna ERR! lerna The latest release of @golevelup/nestjs-discovery is "4.0.1".
lerna ERR! lerna 
lerna ERR! lerna If you need the full list of all 12 published versions run "$ pnpm view @golevelup/nestjs-discovery versions".
```

> I'm trying to publish but running into some issues with the new PNPM setup.
> 
> ```
> pnpm lerna publish -m 'chore: publish'
> lerna notice cli v8.1.8
> lerna info versioning independent
> lerna info Looking for changed packages since @golevelup/nestjs-rabbitmq@5.5.0
> lerna info getChangelogConfig Successfully resolved preset "conventional-changelog-angular"
> 
> Changes:
>  - rabbitmq-integration: 3.4.0 => 3.5.0 (private)
>  - @golevelup/nestjs-common: 2.0.0 => 2.0.1
>  - @golevelup/nestjs-discovery: 4.0.1 => 4.0.2
>  - @golevelup/nestjs-graphql-request: 0.2.2 => 0.2.3
>  - @golevelup/nestjs-hasura: 3.0.7 => 3.0.8
>  - @golevelup/nestjs-rabbitmq: 5.5.0 => 5.6.0
>  - @golevelup/nestjs-stripe: 0.8.2 => 0.9.0
>  - @golevelup/ts-jest: 0.5.6 => 0.6.0
>  - @golevelup/ts-sinon: 0.1.0 => 0.1.1
>  - @golevelup/ts-vitest: 0.5.0 => 0.5.1
>  - @golevelup/nestjs-webhooks: 0.2.18 => 0.2.19
> 
> ? Are you sure you want to publish these packages? Yes
> lerna info execute Skipping releases
> lerna ERR! Error: Command failed with exit code 1: pnpm install --lockfile-only --ignore-scripts
> lerna ERR! Scope: all 13 workspace projects
> lerna ERR!  WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
> lerna ERR! Progress: resolved 1, reused 0, downloaded 0, added 0
> lerna ERR! /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request:
> lerna ERR!  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @golevelup/nestjs-discovery@^4.0.2
> lerna ERR! 
> lerna ERR! This error happened while installing a direct dependency of /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request
> lerna ERR! 
> lerna ERR! The latest release of @golevelup/nestjs-discovery is "4.0.1".
> lerna ERR! 
> lerna ERR! If you need the full list of all 12 published versions run "$ pnpm view @golevelup/nestjs-discovery versions".
> lerna ERR!     at makeError (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/lib/error.js:59:11)
> lerna ERR!     at handlePromise (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/index.js:114:26)
> lerna ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
> lerna ERR!     at async VersionCommand.updatePackageVersions (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/lerna@8.1.8_encoding@0.1.13/node_modules/lerna/dist/index.js:9857:11)
> lerna ERR!     at async Promise.all (index 0)
> lerna ERR! Error: Command failed with exit code 1: pnpm install --lockfile-only --ignore-scripts
> lerna ERR! Scope: all 13 workspace projects
> lerna ERR!  WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
> lerna ERR! Progress: resolved 1, reused 0, downloaded 0, added 0
> lerna ERR! /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request:
> lerna ERR!  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @golevelup/nestjs-discovery@^4.0.2
> lerna ERR! 
> lerna ERR! This error happened while installing a direct dependency of /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request
> lerna ERR! 
> lerna ERR! The latest release of @golevelup/nestjs-discovery is "4.0.1".
> lerna ERR! 
> lerna ERR! If you need the full list of all 12 published versions run "$ pnpm view @golevelup/nestjs-discovery versions".
> lerna ERR!     at makeError (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/lib/error.js:59:11)
> lerna ERR!     at handlePromise (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/execa@5.0.0/node_modules/execa/index.js:114:26)
> lerna ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
> lerna ERR!     at async VersionCommand.updatePackageVersions (/home/jesse/code/personal/golevelup/nestjs/node_modules/.pnpm/lerna@8.1.8_encoding@0.1.13/node_modules/lerna/dist/index.js:9857:11)
> lerna ERR!     at async Promise.all (index 0)
> lerna ERR! lerna Command failed with exit code 1: pnpm install --lockfile-only --ignore-scripts
> lerna ERR! lerna Scope: all 13 workspace projects
> lerna ERR! lerna  WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
> lerna ERR! lerna Progress: resolved 1, reused 0, downloaded 0, added 0
> lerna ERR! lerna /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request:
> lerna ERR! lerna  ERR_PNPM_NO_MATCHING_VERSION  No matching version found for @golevelup/nestjs-discovery@^4.0.2
> lerna ERR! lerna 
> lerna ERR! lerna This error happened while installing a direct dependency of /home/jesse/code/personal/golevelup/nestjs/packages/graphql-request
> lerna ERR! lerna 
> lerna ERR! lerna The latest release of @golevelup/nestjs-discovery is "4.0.1".
> lerna ERR! lerna 
> lerna ERR! lerna If you need the full list of all 12 published versions run "$ pnpm view @golevelup/nestjs-discovery versions".
> ```

I will take a look at it :)

--- a/packages/graphql-request/package.json
+++ b/packages/graphql-request/package.json
@@ -38,9 +38,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@golevelup/nestjs-common": "^2.0.0",
-    "@golevelup/nestjs-discovery": "^4.0.1",
-    "@golevelup/nestjs-modules": "^0.7.1",
+    "@golevelup/nestjs-common": "workspace:^",
+    "@golevelup/nestjs-discovery": "workspace:^",
+    "@golevelup/nestjs-modules": "workspace:^",
     "graphql": "^16.8.1",
     "graphql-request": "^6.1.0"
   },

--- a/packages/hasura/package.json
+++ b/packages/hasura/package.json
@@ -36,9 +36,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@golevelup/nestjs-common": "^2.0.0",
-    "@golevelup/nestjs-discovery": "^4.0.1",
-    "@golevelup/nestjs-modules": "^0.7.1",
+    "@golevelup/nestjs-common": "workspace:^",
+    "@golevelup/nestjs-discovery": "workspace:^",
+    "@golevelup/nestjs-modules": "workspace:^",
     "@hasura/metadata": "^1.0.2",
     "js-yaml": "^4.1.0",
     "zod": "^3.23.8"

--- a/packages/rabbitmq/package.json
+++ b/packages/rabbitmq/package.json
@@ -34,9 +34,9 @@
     "url": "https://github.com/golevelup/nestjs/issues"
   },
   "dependencies": {
-    "@golevelup/nestjs-common": "^2.0.0",
-    "@golevelup/nestjs-discovery": "^4.0.1",
-    "@golevelup/nestjs-modules": "^0.7.1",
+    "@golevelup/nestjs-common": "workspace:^",
+    "@golevelup/nestjs-discovery": "workspace:^",
+    "@golevelup/nestjs-modules": "workspace:^",
     "amqp-connection-manager": "^4.1.14",
     "amqplib": "^0.10.4",
     "lodash": "^4.17.21"

--- a/packages/stripe/package.json
+++ b/packages/stripe/package.json
@@ -36,9 +36,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@golevelup/nestjs-common": "^2.0.0",
-    "@golevelup/nestjs-discovery": "^4.0.1",
-    "@golevelup/nestjs-modules": "^0.7.1"
+    "@golevelup/nestjs-common": "workspace:^",
+    "@golevelup/nestjs-discovery": "workspace:^",
+    "@golevelup/nestjs-modules": "workspace:^"
   },
   "devDependencies": {
     "stripe": "^17.1.0"

--- a/packages/webhooks/package.json
+++ b/packages/webhooks/package.json
@@ -38,8 +38,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@golevelup/nestjs-common": "^2.0.0",
-    "@golevelup/nestjs-modules": "^0.7.1"
+    "@golevelup/nestjs-common": "workspace:^",
+    "@golevelup/nestjs-modules": "workspace:^"
   },
   "peerDependencies": {
     "body-parser": "^1.20.3"

